bokeh/protocol/messages/pull_doc_req.py

```diff
@@ -36,26 +39,26 @@
 # Dev API
 #-----------------------------------------------------------------------------
 
-class pull_doc_req(Message):
+class pull_doc_req(Message[Empty]):
     ''' Define the ``PULL-DOC-REQ`` message for requesting a Bokeh server reply
     with a new Bokeh Document.
 
     The ``content`` fragment of for this message is empty.
 
     '''
 
-    msgtype   = 'PULL-DOC-REQ'
+    msgtype = 'PULL-DOC-REQ'
 
     @classmethod
-    def create(cls, **metadata):
+    def create(cls, **metadata: Any) -> pull_doc_req:
         ''' Create an ``PULL-DOC-REQ`` message
 
         Any keyword arguments will be put into the message ``metadata``
         fragment as-is.
 
         '''
         header = cls.create_header()
-        return cls(header, metadata, {})
+        return cls(header, metadata, Empty())
```

Comment:
`Empty` is too general. Empty dict, tuple, string? If we need "empty" things in other contexts we will end up in the situation where there is one general name (that means something specific) and later-on more specific names, which is always an annoying inconsistency. This type should be named something more specific from the start.